nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -55,6 +55,10 @@ let
     };
   };
 
+  # Load the shipped.json from the Nextcloud package to get alwaysEnabled apps
+  shippedJson = builtins.fromJSON (builtins.readFile "${overridePackage}/core/shipped.json");
```

Comment:
```suggestion
  shippedJson = lib.importJSON "${overridePackage}/core/shipped.json";
```

Reply:
this looks like IFD, which is forbidden in nixpkgs

Reply:
well I was thinking about extending `assertions` with the nix-var later in a new pull ... but as IFD is a hard forbidden thing ... I will not do it